src/arch/arm/machine/l2c_310.c

```diff
@@ -340,6 +340,7 @@ BOOT_CODE void initL2Cache(void)
     /* Direct register access */
     l2cc->control.control |= CTRL_CTRL_EN;
 #endif /* TI_MSHIELD */
+    dsb();
```

Comment:
What do you think about adding line comments on each of these mem barriers explaining their assumptions/guarantees?
Eg:
"The write to the control register needs to complete before the following write to the aux control register.
The dsb guarantees that the setACTLR() doesn't start executing until the completion of the l2cc control register update".

Reply:
I think we should try and aim for comments on all mem barriers used throughout the kernel if we want to be able to keep improving their application.